### DISCUSSION TITLE
group packages and disable only minor and major updates but receive patch updates

### DISCUSSION THREAD

**Discussion by JanMalte** [renovatebot/renovate#21540] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.160.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

15.10.1

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I try to group `contao/.*-bundle` packages and disable `minor` and `major` updates for them, while still receiving `patch` updates.

### What you tried so far.

I tried the following renovate.json but this disabled all updates.

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["local>gegl-group/renovate-config"],
  "packageRules": [
    {
      "description": "Group Contao CMS bundle packages",
      "groupName": "Contao",
      "matchPackagePatterns": ["^contao/.*-bundle"],
      "matchManagers": ["composer"]
    },
    {
      "description": "disable minor and major contao updates to stay with LTS release",
      "matchPackagePatterns": ["^contao/.*-bundle"],
      "matchManagers": ["composer"],
      "matchUpdateTypes": ["major", "minor"],
      "enabled": false
    }
  ],
}
```

Minimal reproduction repository: https://github.com/JanMalte/renovate-contao-group-test

**Comment by rarkins** [comment, marked as answer]
Add `separateMinorPatch=true` to your config, or to your first package rule
